ksqldb-engine/src/test/java/io/confluent/ksql/integration/SecureIntegrationTest.java

```diff
@@ -175,6 +176,7 @@ public void after() {
   }
 
   @Test
+  @Ignore
```

Comment:
This is failing to create the topic and might be an issue. I have no idea why though

Reply:
It seems that query is getting created and not until the `after` the query is getting added and started. However when its being closed we hit 
```[2022-03-03 16:09:18,143] INFO Principal = User:ANONYMOUS is Denied Operation = Delete from host = 192.168.1.97 on resource = Group:LITERAL:_confluent-ksql-default_query_0 for request = OffsetDelete with resourceRefCount = 1 (kafka.authorizer.logger:614)
```

Reply:
@rodesai Do you have an idea of what is causing this?

Reply:
It looks like the test is trying to delete offsets for the group as part of running the query. Seems like it could be a legitimate bug to me. Have you tried setting a breakpoint in `deleteConsumerGroupOffsets` and seeing what's calling that?

Reply:
We delete offsets when closing a query. We need to inorder to make that runtime usable again. What other permissions do we need for that other than the SSL the test is giving us?

Reply:
not quite sure - but you need to configure ACLs explicitly. Can you try changing the line that grants describe/read acls on groups to:
```
givenAllowAcl(ALL_USERS,
                  resource(GROUP, ResourcePattern.WILDCARD_RESOURCE),
                  ops(DESCRIBE, READ, DELETE));
```

Reply:
Yeah that fixed the test. Do we need to make sure that permission is given or is this just a test issue?